pkg/netpol/eval/check.go

```diff
@@ -257,12 +262,12 @@ func (pe *PolicyEngine) allowedXgressConnection(src, dst k8s.Peer, isIngress boo
 		if dst.PeerType() == k8s.IPBlockType {
 			return true, nil // all connections allowed - no restrictions on ingress to externalIP
 		}
```

Comment:
This 'if is redundant - this check was moved to `getPoliciesSelectingPod`

Reply:
removed